### ISSUE TITLE
Value parameter of input_submit not working

### ISSUE THREAD

**Issue by chrilleferna** [formr/Formr#11]
Hi Tim,
Thanks again for the excellent Formr!
I have a small problem with the value parameter for input_submit.

I have the following in my code:
`$form->input_submit('submit', '', 'Enregister', 'submit_id', 'class="btn btn-success"');`

I would expect to see 'Enregistrer' as the the value attribute (and displayed as the text inside the button), but this results in:

`<input type="submit" name="submit" value="Submit" id="submit_id" class="btn btn btn-success">`

So I can't change the submit button's text independently of the button name. Would be nice if you could fix it.

Cheers,
 -- christer

**Comment by chrilleferna** [comment]
I just found out that if I use a name parameter which is different from 'submit' things work. This work-around is fine for me, but you may still want to fix it either in the code or the documentation.

**Comment by chrilleferna** [comment]
Hi again,
There is a similar problem with input_hidden. It takes the label parameter as value. I discovered this when I changed an input_text to input_hidden and my form stopped working as it should

**Comment by timgavin** [comment]
I copy/pasted your code

```
echo $form->input_submit('submit', '', 'Enregister', 'submit_id', 'class="btn btn-success"');
```

And this is what I get.

```
<input type="submit" name="submit" value="Enregister" id="submit_id" class="btn btn btn-success">
```

<img width="808" alt="enregister" src="https://user-images.githubusercontent.com/1012049/58037709-e1bd7c00-7ae2-11e9-866f-ed79a65b74fb.png">

Works as expected, so perhaps you have something else going on somewhere?

**Comment by chrilleferna** [comment]
Thanks a lot for your quick reply!  My code has changed quite a bit since when I wrote my question earlier today and in fact the first case now works as expected. Can't really understand what happended. 
Concerning the second case, when changing an input_text to input_hidden, I just saw in your documentation that the value parameter, which is the third parameter in input_text is the second parameter in input_hidden. So in fact things behave as described in the documentation, although it is a little bit unexpected.
